### PR TITLE
whipper: replace morituri

### PR DESCRIPTION
**Pull request by jtojnar**
###### Motivation for this change
morituri has been dead for a while now and uses gst-python which is [no longer supported](https://bugzilla.gnome.org/show_bug.cgi?id=797138) wth Python 2. whipper is a maintained fork, packaged, for example, in Arch.

cc @rycee @jgeerds @jansol 

Closes: https://github.com/NixOS/nixpkgs/issues/27228

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by jtojnar** [comment]
Forgot to add `accuraterip-checksum` expression. Please try again.

File: pkgs/tools/audio/accuraterip-checksum/default.nix [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  installPhase = ''
+    runHook preInstall
+
+    install -D -m755 accuraterip-checksum "$out/bin/accuraterip-checksum"
+
+    runHook postInstall
+  '';
+
+  meta = with stdenv.lib; {
+    description = "A C99 commandline program to compute the AccurateRip checksum of singletrack WAV files";
```
Comment (rycee) [inline pkgs/tools/audio/accuraterip-checksum/default.nix]: I think this description could be simplified to "Computes the AccurateRip checksum of singletrack WAV files".

File: pkgs/top-level/all-packages.nix [hunk 2]
```diff
@@ -3613,7 +3615,7 @@ with pkgs;
 
   mmv = callPackage ../tools/misc/mmv { };
 
-  morituri = callPackage ../applications/audio/morituri { };
+  whipper = callPackage ../applications/audio/whipper { };
```
Comment (rycee) [inline pkgs/top-level/all-packages.nix]: I think `whipper` should be placed further down the list to maintain the "pseudo-alphabetic" ordering of the file.

**Comment by rycee** [comment]
Thanks @jtojnar, looks good to me. I just added a few minor comments.

**Comment by rycee** [comment]
Nice! Merged to master :+1:

**Pull request merged**